io/net/ethtool_test.py

```diff
@@ -59,7 +59,11 @@ def setUp(self):
         if device in interfaces:
             self.interface = device
         elif local.validate_mac_addr(device) and device in local.get_all_hwaddr():
-            self.interface = local.get_interface_by_hwaddr(device).name
+            if self.hbond:
+                self.interface = local.get_interface_by_hwaddr(device).name
+                self.interface = self.interface._get_bondingmaster()
```

Comment:
do you need this extra line `self.interface = local.get_interface_by_hwaddr(device).name` ?